test/unit/behaviors.html

```diff
@@ -53,13 +53,17 @@
   });
 
   test('beforeRegister behavior', function() {
-    var el = document.createElement('behavior-as-is');
-    assert.equal(el.is, 'behavior-as-is');
-    assert.ok(el.$.content);
-    el.prop = 42;
-    assert.isTrue(el.propChanged1Called);
-    assert.isTrue(el.propChanged2Called);
-    assert.isTrue(el.hasAttribute('attr'));
+    if (Polymer.Settings.lazyRegister === 'max') {
+      assert.isTrue(window.behaviorAsIsFail, 'Setting `is` in a behavior did not fail when lazyRegister: `max` set');
+    } else {
```

Comment:
Sorry, got it.  `is` is defined in a behavior, which is the thing not supported with `max` mode.